pkg/cosign/kubernetes/webhook/validator.go

```diff
@@ -314,7 +314,7 @@ func validatePolicies(ctx context.Context, ref name.Reference, policies map[stri
 // signatures OR attestations if atttestations were specified.
 // Returns PolicyResult, or errors encountered if none of the authorities
 // passed.
-func ValidatePolicy(ctx context.Context, ref name.Reference, cip webhookcip.ClusterImagePolicy, remoteOpts ...ociremote.Option) (*PolicyResult, []error) {
+func ValidatePolicy(ctx context.Context, namespace string, ref name.Reference, cip webhookcip.ClusterImagePolicy, remoteOpts ...ociremote.Option) (*PolicyResult, []error) {
```

Comment:
Nit, we should document in the signature what namespace this is. It's the namespace where we allow the signature pull secrets to come from.

Reply:
+1